github-actions/src/main/scala/org/typelevel/sbt/gha/GenerativePlugin.scala

```diff
@@ -64,6 +67,17 @@ object GenerativePlugin extends AutoPlugin {
 
   import autoImport._
 
+  private object MatrixKeys {
+    val OS = "os"
+    val Scala = "scala"
+    val Java = "java"
```

Comment:
Stupid micro-optimization 😜 adding `final` means the compiler will inline them at use-site.
```suggestion
    final val OS = "os"
    final val Scala = "scala"
    final val Java = "java"
```